### PR TITLE
Add project name to user-data

### PR DESCRIPTION
**Pull request by jconstance-amplify**
It would be convenient for some runtime scripts to know the name of the project without having to poke around at asiaq config files. This seems pretty harmless and a lot easier to work with.

### CODE REVIEW

**Comment by jconstance-amplify** [comment]
@pchand-amplify @IlyaSukhanov @aoskotsky-amplify

**Comment by pchand-amplify** [comment]
What's the use case?

**Comment by jconstance-amplify** [comment]
I don't want to hardcode ```mclass``` here:

https://github.com/amplifylitco/asiaq_reporting_config/pull/48/files#diff-27354667902e6cd08c1226d98758b026R55

File: disco_aws_automation/disco_aws.py [hunk 1]
```diff
@@ -162,6 +162,7 @@ def create_userdata(self, hostclass, owner, testing):
                 data["route_table_id_{0}".format(_)] = _tmp[_][0]
                 data["destination_cidr_block_{0}".format(_)] = _tmp[_][1]
         logging.debug("userdata: %s", data)
+        data["project_name"] = self.config("project_name")
```
Comment (pchand-amplify) [inline disco_aws_automation/disco_aws.py]: Before sending to the hostclass we should log it for debugging purposes.

**Comment by pchand-amplify** [comment]
Once the fallback is addressed :+1:

**Comment by IlyaSukhanov** [comment]
+1

File: disco_aws_automation/disco_aws.py (hunk 1 above)
Comment (jconstance-amplify) [inline disco_aws_automation/disco_aws.py]: Whoops. I'm assuming should be fine to just swap the new line and the logging line.

Comment (pchand-amplify) [inline disco_aws_automation/disco_aws.py]: :+1:

**Pull request merged**